src/backend/utils/misc/queryjumble.c

```diff
@@ -597,6 +597,24 @@ JumbleExpr(JumbleState *jstate, Node *node)
 				JumbleExpr(jstate, rtfunc->funcexpr);
 			}
 			break;
+		case T_Integer:
+			{
+				APP_JUMB(intVal(node));
+			}
+			break;
+		case T_GroupingFunc:
+			{
+				GroupingFunc *gf = (GroupingFunc *) node;
+				JumbleExpr(jstate, (Node *) gf->args);
```

Comment:
Maybe we still need to backport the [commit](https://github.com/postgres/postgres/commit/f7a97b6ec31f3f57a6154d0039c4de81ad517064)?

Reply:
the same issue with GroupingFunc struct - in adb-6 there are no such fields that were used in mentioned commit.